05-Jason-Luo/wk07/2-tue/dogs/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/water.css@2/out/water.min.css"
+    />
+    <title>Dogs</title>
+  </head>
+  <body>
+    <h1>Good Pupper</h1>
+
+    <button id="fetch">Look for Good Boy</button>
+    <div id="root"></div>
+
+    <script src="js/jquery.js"></script>
+    <script src="js/dogs.js"></script>
```

Comment:
Again, try to move these up into the `<head>` tag. That's where non-content based tags ought to go.